Assignment_Git_Problem1-4.R

```diff
@@ -0,0 +1,139 @@
+#BAN400 Assignment Git
+
+#Problem 1 -4
+
+##############################################################################
+#Problem1 
+# I did it by clone, update the local file, add and commit, and push
+
+
+##############################################################################
+#Problem2
+# read data "suites_dw_Table1.txt" to a tidy data frame in R, 
+# without changing the raw data file and without hard coding line numbers.
+
+#set working directory
+
+#load packages
+library(tidyverse)
+library(ggplot2)
+
+#Read text data
+raw_text <- readLines(con = "C:/Users/littl/ban400/git-Tomoko-Yamagata/suites_dw_Table1.txt")
+
+#Convert the text file to data frame
+#separate with "|"
+#skip first 14 rows
+
+df_galaxy<-
+  read.delim("suites_dw_Table1.txt",
+           header = FALSE,
+           sep = "|",
+           skip = 14
+           ) 
+
+#Add column's names from row 13
+colnames(df_galaxy)<-
+  readLines("suites_dw_Table1.txt")[13] %>%
+  strsplit(split = "\\|") %>% unlist()
+
```

Comment:
Very nicely done. Dataframe looks good.

Reply:
There are some unnecessary whitespace in the column names now. You can use this to remove those:

```
colnames(df_galaxy2) <- sapply(colnames(df_galaxy2), trimws)
```